docker/bugzilla.conf

```diff
@@ -11,6 +11,7 @@ ServerName localhost:80
         DirectoryIndex index.cgi
         Options Indexes FollowSymLinks ExecCGI
         AllowOverride All
-        Require all granted
+        Satisfy Any
+        Allow from all
     </Directory>
```

Comment:
We should just grab the real http conf from SCL3 webheads and use it. Minus the SSL certs, etc.